src/beam.ts

```diff
@@ -74,6 +76,15 @@ export class Beam extends Element {
   private beam_count: number;
   private unbeamable?: boolean;
 
+  /**
+   * Overrides to default beam directions for secondary-level beams that do not
+   * connect to any other note. See further explanation at
+   * `setPartialBeamSideAt`
+   */
+  private forced_partial_directions: {
```

Comment:
Can we make this camelCase (i.e. `forcedPartialDirection`)?

Same with beam_count => beamCount above. I think JS style guides generally avoid underscores in variable names, except with UPPER_CASE_CONSTANTS.

Reply:
I just realized beam_count isn't part of your PR, haha... and it looks like beam.ts has a lot of underscore_variables already. I do think new contributions should move toward camelCase, and it's possible that in some future (breaking) VexFlow, we'll move everything to camelCase.

Reply:
Yes, tons of snake_case in beam.ts currently. Most of them are for private class members. Shall I go ahead and convert those as well?

Reply:
We could convert them in a subsequent PR. It'll make this one easier to review/test. I'm not a music expert, but the new feature looks cool!

Reply:
Ok, great -- I'll change the snake_cases that I added to camelCase and commit, but leave the existing variables/class members unchanged. Let me know if there's anything else you see that needs attention. Thanks!